core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -36,6 +37,12 @@ object Logging {
       } yield ()
     )
 
+  def error(cause: Cause[Any]): ZIO[Logging, Nothing, Unit] =
+    log(LogLevel.Error)(cause.prettyPrint)
+
+  def throwable(t: Throwable): ZIO[Logging, Nothing, Unit] =
```

Comment:
I would actually implement this one by wrapping the `Throwable` into a `Cause`, and then delegating to `error`.

Reply:
done